resolwe/flow/tests/test_executors.py

```diff
@@ -85,10 +85,12 @@ def test_spawn(self):
         # Check correct dependency type is created.
         self.assertEqual({d.kind for d in data.parents_dependency.all()}, {DataDependency.KIND_SUBPROCESS})
 
+    @unittest.skipIf(True, 'since PR308: the exception happenes in other processes, currently impossible to propagate')
     def test_spawn_missing_export(self):
         with six.assertRaisesRegex(self, KeyError, 'Use `re-export`'):
             self.run_process('test-spawn-missing-file')
 
+    @unittest.skipIf(True, 'since PR308: impossible to propagate error information at the moment')
```

Comment:
This test should work?

Reply:
Yes, it does. The unconditional skip was from much earlier, and a later fix made it possible for it to work; I neglected to test it.